docs/Implementation.tex

```diff
@@ -40,6 +40,21 @@
 \section{Einleitung}
 
 
+\section{Workflow}
+OSIP wurde von uns open-source mit Hilfe von GitHub entwickelt. Um die Qualität des Codes möglichst gut zu halten,
+haben wir konsequent Code-Reviews eingesetzt. Jede Änderung musste von einem Kommilitonen genehmigt werden. Zusätzlich ließen wir
+durch den Dienst "`Travis"' bei jeder einzubindenden Änderung die Stil-Richtlinien für den Code (mittels Checkstyle) und die Unit Tests überprüfen.
+Dieses Vorgehen erforderte zwar oft lange Wartezeiten auf Reviews, im Sinne der guten Qualität haben wir uns allerdings darauf eingelassen. 
+
+Um die Aufgaben der einzelnen Kommilitonen aufeinander abzustimmen, setzten wir den von GitHub bereitgestellten Issue-Tracker ein. Im Laufe der
+Entwicklung von OSIP wurden über 400 Issues und Pull requests erstellt.
+
+\section{Arbeitsaufteilung}
```

Comment:
Haben wir nicht eher Outside-In gemacht, weil wir gleichzeitig mit View und Model angefangen haben? Au0erdem würde ich dazu schreiben, warum wir das so gemacht haben (also weil sich Niko und Martin mit der aufwändigen View auskannten, und wir anderen so lange das Model machen konnten, bis der Großteil des Views implementiert war)